src/R_popup.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable */
+
```

Comment:
- [ ] Kindly enable the linter for this file and remove all linter disables. Linters enforces code quality and therefore should not be disabled. Instead, you should correct the issues raised.

Reply:
Not addressed